docs/notebooks/dispersion/two_fluid_dispersion_solver.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<p>A couple of you lines are too long and a few of the indentations don't follow PEP8 standards.  Nothing else is wrong, the formatting just needs to be cleaned up a bit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/932/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
In the most recent commit, I have updated the lines so that it is consistent with PEP8 standards and lines are of proper length. Can you please check it once and confirm that that's the case?

Reply:
Also...in the documentation part I can't figure out how to display the title and it shows as "no title", can you please look into this as well!